tests/unit/lib/WUI/http/multipart_upload.cpp

```diff
@@ -230,11 +230,11 @@ TEST_CASE("One Big Chunk") {
 
     const auto &log = test.log();
     REQUIRE(log.start.has_value());
-    REQUIRE(log.start->filename == "tmp.gcode");
+    REQUIRE(log.start->filename == "upload.tmp");
```

Comment:
may be declare a constant (other than DEFAULT_UPLOAD_FILENAME) for the unit tests as well

Reply:
I'll merge it this way. I know this is a bit of slacky, but that's a bit of a reminder about another TODO ‒ these names need to be generated dynamically in the future to allow parallel uploads.